docs/usage/self-hosted-configuration.md

```diff
@@ -376,9 +376,18 @@ In practice, it is implemented by converting the `force` configuration into a `p
 This is set to `true` by default, meaning that any settings (such as `schedule`) take maximum priority even against custom settings existing inside individual repositories.
 It will also override any settings in `packageRules`.
 
-## forkToken
+## forkCreate
+
+Set this to false if using an app token which is not permitted to fork.
+
+## forkOrgs
 
-You probably don't need this option - it is an experimental setting developed for the Forking Renovate hosted GitHub App.
+Configuring this value lets you choose which org/account you want repos forked into when fork mode is enabled.
+It also allows you to configure more than one if you are migrating from user-based forks into org-based ones.
+
+Renovate will first look if a fork exists in the preferred org, and then if not then look to see if one exists in any of the fallback orgs.
```

Comment:
```suggestion
This configuration option lets you choose which organization or account you want repositories forked into when "fork mode" is enabled.

You may set more than one organization or account.
This can be handy if you're migrating from user-based forks to organization-based forks.

If you've set multiple `forkOrgs` then Renovate will:

1. Check if a fork exists in the preferred organization
1. If no fork exists in the preferred org: Renovate checks the fallback organizations

Renovate always creates the new fork in the _first_ organization in the `forkOrgs` list.
```

How about this wording and layout?

> maybe add one more sentence, that renovate will create new fork in first org of the list?

My suggestion adds this sentence. 😉